Makefile

```diff
@@ -38,13 +38,15 @@ endif
 gresource: .gresource.xml icon2.png
 	glib-compile-resources .gresource.xml --generate-source --target=src/resources.c
 
+droidcam-cli: LDLIBS += $(JPEG) $(LIBAV) $(LIBS)
```

Comment:
Why re-declare the target, instead of adding as a step in the existing target?

Reply:
Because I'm declaring a target-specific variable: https://www.gnu.org/software/make/manual/make.html#Target_002dspecific